src/components/Article.jsx

```diff
@@ -30,7 +30,7 @@ export default function Article() {
       setVotes((currentVotes) => {
         return currentVotes - 1;
       });
-      console.log(err);
+      console.log("Upvoting was not possible at this time");
```

Comment:
You will only be able to see this message in the console, but we want to ensure our user can see it on the UI - so in this case, we would need to return the message in some sort of hmtl tag.